mantidimaging/gui/windows/operations/filter_previews.py

```diff
@@ -199,22 +197,3 @@ def set_histogram_log_scale(self):
         """
         set_histogram_log_scale(self.image_before_hist)
         set_histogram_log_scale(self.image_after_hist)
-
-    def _add_auto_colour_action(self, img_view: MIMiniImageView):
```

Comment:
I there are a few things that are not used and can now be cleaned up.
self.all_histograms

Also now is probably a good time to remove the 
`self.image_before, self.image_before_vb, self.image_before_hist = self.imageview_before.get_parts()`
These were to ease the transition to MIMiniImageView. Now that there are only a few places that directly access these, they can used the MIMiniImageView accessors.

e.g. use of`self.image_after_vb` would become `self.imageview_after.viewbox`

Same changes can be made in `gui/windows/recon/image_view.py`

Then get_parts can be removed from MIMiniImageView.